tests/test_volume.py

```diff
@@ -358,6 +417,39 @@ def test_rotate(L, dtype):
         assert np.allclose(ref_vol, rot_vol, atol=utest_tolerance(dtype))
 
 
+def test_rotate(vols_hot_cold):
+    """
+    We rotate Volumes containing random hot/cold spots by random rotations and check that
+    hot/cold spots in the rotated Volumes are in the expected locations.
+    """
+    vols, hot_cold_locs, vol_center = vols_hot_cold
+    dtype = vols.dtype
+    L = vols.resolution
+
+    # Generate random rotations.
+    rots = Rotation.generate_random_rotations(n=vols.n_vols, dtype=dtype)
+    rots_mat = rots.matrices
+
+    # Expected location of hot/cold spots relative to (0, 0, 0) origin in (x, y, z) order.
+    expected_hot_cold = np.transpose(
```

Comment:
similar to before, we may be able to avoid the double transpose here.

Reply:
Done. Thanks